lib/gltfPipeline.js

```diff
@@ -96,11 +97,12 @@ function processFile(inputPath, options, callback) {
 }
 
 function writeFile(gltf, outputPath, options, callback) {
+    var fileExtension = path.extname(outputPath);
```

Comment:
I don't know how all of this is wired together, but can't we just make sure that `binary` is set to `true` before this is called; otherwise, this is forcing some of the command-line logic deeper in the engine.